runelite-client/src/main/java/net/runelite/client/plugins/timers/GameTimer.java

```diff
@@ -38,49 +38,53 @@
-	SNARE("snare", GraphicID.SNARE, 10, ChronoUnit.SECONDS),
-	HALFSNARE("snare", GraphicID.SNARE, 5, ChronoUnit.SECONDS),
-	ENTANGLE("entangle", GraphicID.ENTANGLE, 15, ChronoUnit.SECONDS),
-	HALFENTANGLE("entangle", GraphicID.ENTANGLE, 7500, ChronoUnit.MILLIS),
-	ICERUSH("icerush", GraphicID.ICE_RUSH, 5, ChronoUnit.SECONDS),
-	ICEBURST("iceburst", GraphicID.ICE_BURST, 10, ChronoUnit.SECONDS),
-	ICEBLITZ("iceblitz", GraphicID.ICE_BLITZ, 15, ChronoUnit.SECONDS),
-	ICEBARRAGE("icebarrage", GraphicID.ICE_BARRAGE, 20, ChronoUnit.SECONDS),
-	IMBUEDHEART("imbuedheart", GraphicID.IMBUED_HEART, 420, ChronoUnit.SECONDS),
-	VENGEANCE("vengeance", GraphicID.VENGEANCE, 30, ChronoUnit.SECONDS);
+	STAMINA("stamina", "Stamina", 2, ChronoUnit.MINUTES),
+	ANTIFIRE("antifire", "Antifire", 6, ChronoUnit.MINUTES),
+	EXANTIFIRE("exantifire", "Extended Antifire", 12, ChronoUnit.MINUTES),
+	EXSUPERANTIFIRE("exsuperantifire", "Extended Super Antifire", 6, ChronoUnit.MINUTES),
+	OVERLOAD("overload", "Overload", 5, ChronoUnit.MINUTES),
+	CANNON("cannon", "Cannon", 25, ChronoUnit.MINUTES),
+	MAGICIMBUE("magicimbue", "Magic Imbue", 15, ChronoUnit.SECONDS),
+	FULLTB("teleblock", "Full Telebock", 5, ChronoUnit.MINUTES),
+	HALFTB("teleblock", "Half Telebock", 150, ChronoUnit.SECONDS),
+	SUPERANTIVENOM("antivenom", "Anti-venom++", 3, ChronoUnit.MINUTES),
+	SUPERANTIFIRE("superantifire", "Super Antifire", 2, ChronoUnit.MINUTES),
+	ANTIDOTEPLUSPLUS("antidoteplusplus", "Antidote++", 12, ChronoUnit.MINUTES),
+	BIND("bind", "Bind", GraphicID.BIND, 5, ChronoUnit.SECONDS),
+	HALFBIND("bind", "Half Bind", GraphicID.BIND, 2500, ChronoUnit.MILLIS),
+	SNARE("snare", "Snare", GraphicID.SNARE, 10, ChronoUnit.SECONDS),
+	HALFSNARE("snare", "Half Snare", GraphicID.SNARE, 5, ChronoUnit.SECONDS),
+	ENTANGLE("entangle", "Entangle", GraphicID.ENTANGLE, 15, ChronoUnit.SECONDS),
+	HALFENTANGLE("entangle", "Half Entangle", GraphicID.ENTANGLE, 7500, ChronoUnit.MILLIS),
+	ICERUSH("icerush", "Ice Rush", GraphicID.ICE_RUSH, 5, ChronoUnit.SECONDS),
+	ICEBURST("iceburst", "Ice Burst", GraphicID.ICE_BURST, 10, ChronoUnit.SECONDS),
+	ICEBLITZ("iceblitz", "Ice Blitz", GraphicID.ICE_BLITZ, 15, ChronoUnit.SECONDS),
+	ICEBARRAGE("icebarrage", "Ice Barrage", GraphicID.ICE_BARRAGE, 20, ChronoUnit.SECONDS),
+	IMBUEDHEART("imbuedheart", "Imbued Heart", GraphicID.IMBUED_HEART, 420, ChronoUnit.SECONDS),
+	VENGEANCE("vengeance", "Vengeance", GraphicID.VENGEANCE, 30, ChronoUnit.SECONDS);
 
 	@Getter
 	private final String imageResource;
 	@Getter
 	private final Duration duration;
 	@Getter
 	private final Integer graphicId;
+	@Getter
+	private final String description;
 
 	private BufferedImage image;
 
-	GameTimer(String imageResource, Integer graphicId, long time, ChronoUnit unit)
+	GameTimer(String imageResource, String description, Integer graphicId, long time, ChronoUnit unit)
 	{
 		this.imageResource = imageResource;
 		this.graphicId = graphicId;
+		this.description = description;
```

Comment:
this.description line should be above graphicsId, since the constructor params have that order